test/lib/pipedprocess.test.js

```diff
@@ -26,7 +26,7 @@ test('execute throws error when exec provides non zero code', () => {
 
 test('execute includes stderr when exec provides non zero code', () => {
     // Execute cat with an arbitrary option
-    expect(() => execute(cat, ['-kj3'], testString)).toThrowError('invalid option');
```

Comment:
was getting an error here trying to get the exact string `invalid option`, so this should work for now:

![image](https://user-images.githubusercontent.com/364566/72634102-bf4c5e80-390e-11ea-8102-7cddeb7f93b6.png)